Neos.Fusion/Tests/Functional/FusionObjects/Fixtures/Fusion/NestedOverwritesAndProcessors.fusion

```diff
@@ -2,15 +2,15 @@ prototype(Neos.Fusion:TestRenderer).@class = 'Neos\\Fusion\\Tests\\Functional\\V
 prototype(Neos.Fusion:Value).@class = 'Neos\\Fusion\\FusionObjects\\ValueImplementation'
 prototype(Neos.Fusion:RawArray).@class = 'Neos\\Fusion\\FusionObjects\\RawArrayImplementation'
 prototype(Neos.Fusion:Attributes).@class = 'Neos\\Fusion\\FusionObjects\\AttributesImplementation'
-prototype(Neos.Fusion:Tag) {
+prototype(Neos.Fusion:LegacyTag) {
 	@class = 'Neos\\Fusion\\FusionObjects\\TagImplementation'
 	attributes = Attributes
```

Comment:
```suggestion
	attributes = Neos.Fusion:Attributes
```

Reply:
I, who come from the future, prophesy that the problem will be solved, but not here and now.